.github/workflows/tests.yml

```diff
@@ -71,3 +68,65 @@ jobs:
     - name: Tests
       run: |
         pytest tests --ignore tests/integration_tests
+
+  coverage:
```

Comment:
How about creating a separate workflow, e.g. `Tests (Coverage)` considering the fact that unit tests are already split depending on whether they're testing integration modules or not. Also, I think we can skip the cron scheduling for the code coverage.